content/en/blog/_posts/2022-10-03-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: blog
+title: "Current State: 2019 Third Party Security Audit of Kubernetes"
+date: 2022-10-03
+slug: current-state-2019-third-party-audit
+---
+
+**Authors**: Cailyn Edwards, Pushkar Joglekar, Rory McCune and Rey Lejano
+
+A brand new Third Party Security Audit of Kubernetes, is expected to be
+published later this month (Oct 2022). 
+
+In preparation for that, let's look at the state of findings that were made
+public as part of the last [third
+party security audit of
+2019](https://github.com/kubernetes/sig-security/tree/main/sig-security-external-audit/security-audit-2019).
+
+## Motivation
+
+Craig Ingram (@cji) has graciously attempted over the years to keep track of the 
```

Comment:
Do we intend to link to this person's GitHub handle?
```suggestion
Craig Ingram (@cji) has graciously attempted over the years to keep track of the 
```

Reply:
Done